nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/AssertEx.cs

```diff
@@ -97,7 +97,7 @@ public static void Equal<T>(
             return;
         }
 
-        Assert.True(false, GetAssertMessage(expected, actual, comparer, message));
+        Assert.Fail(GetAssertMessage(expected, actual, comparer, message));
```

Comment:
Haha, I've made this exact change in other codebases and I smile every time.  "Assert tru.. NO!  FALSE!"